src/components/Error.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { TodosContext } from './TodosContext';
+
+export const Error: React.FC = () => {
+  const { errorMessage, setErrorMessage } = useContext(TodosContext);
+  let isErrorHidden = true;
```

Comment:
generally `isErrorHidden === !errorMessage`, so you can get rid of `isErrorHidden` variable and always use `!errorMessage`, don't you?